docker-compose/ckan-conf-templates/datagov-catalog-production.ini.template

```diff
@@ -0,0 +1,231 @@
+[DEFAULT]
```

Comment:
please modify the main datagov flavour production.ini template and don't create a new flavour.